modules/understanding-upgrade-channels.adoc

```diff
@@ -134,5 +134,21 @@ ifndef::openshift-origin[]
 [discrete]
 == Switching between channels
 
-Your cluster is still supported if you change from the `stable-{product-version}` channel to the `fast-{product-version}` channel. Although you can switch to the `candidate-{product-version}` channel at any time, some releases in that channel might be unsupported release candidates. You can switch from the `candidate-{product-version}` channel to the `fast-{product-version}` channel if your current release is a general availability release. You can always switch from the `fast-{product-version}` channel to the `stable-{product-version}` channel, although if the current release was recently promoted to `fast-{product-version}` there can be a delay of up to a day for the release to be promoted to `stable-{product-version}`. If you change to a channel that does not include your current release, an alert displays and no updates can be recommended, but you can safely change back to your original channel at any point.
+A channel can be switched from the web console or through the `adm upgrade channel` command:
+
+----
+$ oc adm upgrade channel clusterversion version --type json -p '[{"op": "add", "path": "/spec/channel", "value": "<channel>”}]'
```

Comment:
@tsze-redhat Are any other variables in this command? Not sure about the `clusterversion version` part.

Reply:
@opayne1 
Here is an example of the command on an existing cluster:

$ ./oc patch clusterversion version --type json -p '[{"op": "add", "path": "/spec/channel", "value": "stable-4.9"}]'
clusterversion.config.openshift.io/version patched

The command with "clusterversion version" is correct (but does look confusing).

@wking can shed more light on the command if you have more question.

Reply:
LGTM